lldb/source/Plugins/RegisterTypeBuilder/RegisterTypeBuilderClang.cpp

```diff
@@ -43,8 +43,7 @@ CompilerType RegisterTypeBuilderClang::GetRegisterType(
       ScratchTypeSystemClang::GetForTarget(m_target);
   assert(type_system);
 
-  std::string register_type_name = "__lldb_register_fields_";
-  register_type_name += name;
+  std::string register_type_name = "__lldb_register_fields_" + name;
```

Comment:
nit: you could use a `llvm::Twine` here.

Reply:
I'll always need a StringRef for GetTypeForIdentifier, and I don't think the Twine can be represented as a single StringRef in this case. So I'll have to call .str() on the Twine regardless.

Unless I am missing another advantage to Twine here?

Reply:
Yeah, I was thinking making a `Twine` then building a `std::string` out of it. Nvm